### PR TITLE
[MBL-16286][Student][Teacher] - Add timestamp to dev build apks in Student/Teacher apps.

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Add timestamp to qa and debug build apks in Student/Teacher apps.

refs: MBL-16286
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]

**Review by kristofnemere — APPROVED** [review]

File: apps/student/build.gradle [hunk 1]
```diff
@@ -143,6 +143,19 @@ android {
                 abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86_64'
             }
         }
+
+        applicationVariants.all {
```
Comment (hermannakos) [inline apps/student/build.gradle:147]: flank won't be able to find the apk this way and break all the tests, you should look into that as well.

**Review by hermannakos — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request closed without merging**